apps/wfo-ui/pages/metadata/productblocks.tsx

```diff
@@ -1,12 +1,6 @@
 import React from 'react';
-import { MetaDataLayout } from '../../components/Metadata/PageLayout/layout';
+import { WFOProductBlocksPage as Page } from '@orchestrator-ui/orchestrator-ui-components';
```

Comment:
A cast to "Page" does not seem to be needed here, since the router searches for the default export in this file

Reply:
Fixed